src/create-repo-relationship.ts

```diff
@@ -1,82 +1,83 @@
+// match what's in npm (other than the `@` sign).
+// For example, TitleCase vs. all lowercase below:
+// - in Github: `JupiterOne/jupiterone-client-nodejs`
+// - in npm: `@jupiterone/jupiterone-client-nodejs`
+const scopeNameTransformations = new Map([
+  ['@jupiterone/', 'JupiterOne/']
+])
+
+export async function createRepoRelationships(
+  depsList: string[],
+  mainRepo: any[],
+  j1Client,
+  repoName: string,
+  missingDeps: string[],
+  forDeploy?: boolean
+) {
+  const results = {success: 0, failure: 0, missingDeps: []};
+  const depRegex = /^(@\w+\/)?([^:]+)(:.*)?$/;
   if (depsList.length > 0) {
     for (const dep of depsList) {
-      const fullName = dep.substring(1, dep.indexOf(':'));
-      const depRepo = await j1Client.queryV1(
-        `FIND CodeRepo WITH full_name='${fullName}'`
-      );          
+      const match = dep.match(depRegex);
+      if (!match) {
+        continue;
+      }
+
+      const scope = match[1];
+      const repo = match[2];
+
+      let fullName;
+      if (scope && repo) {
+        fullName = scopeNameTransformations.has(scope)
+          ? scopeNameTransformations.get(scope) + repo
+          : scope.substring(1) + repo
+      } 
+      else if (repo) {
+        fullName = repo;
+      }
+      else {
+        continue;
+      }
+      
+      const j1query = scope
+        ? `FIND CodeRepo WITH full_name='${fullName}'`
+        : `FIND CodeRepo WITH name='${fullName}'`
+      const depRepo = await j1Client.queryV1(j1query);          
       
       if (depRepo.length === 1) {
         const relationshipKey = mainRepo[0].entity._key + '|uses|' + depRepo[0].entity._key;
         const relationship = await j1Client.createRelationship(
           relationshipKey,
-          'bitbucket_repo_uses_bitbucket_repo',
+          'repo_dependency',
           'USES',
           mainRepo[0].entity._id,
           depRepo[0].entity._id,
         );
-        console.log('Successfully created relationship (' + repoName + ' USES ' + dep + ').');
-        successFail.success++;
-      } else if (depRepo.length > 1) {
-        console.log('Failed to create relationship with ' + dep + 
-        ' (query returned multiple results). Skipped.' )
-      }
-      else {
         console.log(
-          'Failed to create relationship with ' + dep + 
-          ' (was not found on the graph). Skipped.'
+          `Successfully created relationship (${repoName} USES ${dep}${
+            forDeploy ? ' for deploy' : ''
+          }).`
         );
-        if (!missingDeps.includes(dep)) {
-          missingDeps.push(dep + ' (' + repoName + ').');
-        }
-        successFail.failure++;
-      }
-    }
-  } else {
-    console.log('Repo has no dependencies of requested scopes');
-  }
-  successFail.missingDeps = missingDeps;
-  return successFail;
-}
-
-export async function createDeployRelationships(deployDepsList, mainRepo, j1Client, repoName, missingDeps) {
```

Comment:
i removed this and combined with the function above.  they are pretty much the same.